src/foundation/colors/css.ts

```diff
@@ -1,151 +1,112 @@
 import { css } from '@linaria/core';
 
+export const themedColors = {
+  onbackground1: 'var(--sb-bg-1)',
+  onbackground2: 'var(--sb-bg-2)',
+  onbackground3: 'var(--sb-bg-3)',
+  onbackground4: 'var(--sb-bg-4)',
+  oncontent1: 'var(--sb-content-1)',
+  oncontent2: 'var(--sb-content-2)',
+  oncontent_inverse1: 'var(--sb-content-inverse-1)',
+  oncontent_inverse2: 'var(--sb-content-inverse-2)',
+  primary: 'var(--sb-primary)',
+  error: 'var(--sb-error)',
+  secondary: 'var(--sb-secondary)',
+};
+export const themedColorVars = css`
+  .sendbird-theme--light & {
+    --sb-bg-1: var(--sendbird-light-onlight-01);
```

Comment:
is `--sb-bg-1` good name for `--sendbird-light-onlight-01`. because i bg stands for background but i saw many cases `--sendbird-light-onlight-01` were being used in non background colors such as border color and text color

Reply:
can we just use `--sb-light-1` ?

Reply:
At least in UIKit, there are no such cases. Currently, our primary objective is to directly port the code from UIKit, so we will maintain it as is. If such cases are added in the future, it will not be too late to consider them then.